centreon/packages/ui/src/Form/CollapsibleGroup.tsx

```diff
@@ -66,7 +66,7 @@ const CollapsibleGroup = ({
 
   const containerClassName = className || '';
 
-  const CollapseIcon = isOpen ? ExpandMore : ChevronRightIcon;
+  const CollapseIcon = isOpen ? ExpandLess : ExpandMore;
```

Comment:
This is the endless human dilemma

Reply:
Right, but let's at least have a consistent behavior with the other components